app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
🛠 It would be great to attach a screenshot to the description of the PR so we could see the result of any UI changes.

Reply:
Will make sure to do this on the next PR!